R/estimate_infections.R

```diff
@@ -52,7 +52,7 @@
 #'
 #' # default setting
 #' # here we assume that the observed data is truncated by the same delay as
-#' def <- estimate_infections(reported_cases,
+# ' def <- estimate_infections(reported_cases,
 #'   generation_time = generation_time,
```

Comment:
This line is belongs to `estimate_infections()` in the previous line:
```r
Warning message:
When processing R/estimate_infections.R: <text>:18:36: unexpected ','
17: # ' def <- estimate_infections(reported_cases,
18:   generation_time = generation_time,
```

Reply:
:+1: thanks good spot